docs/user/bots.rst

```diff
@@ -654,9 +654,10 @@ The Cache is required to memorize which files have already been processed (TTL n
 
 **Configuration Parameters**
 
-* `country`: The country you want to download the reports for
+* `country`: **Deprecated:** The country you want to download the reports for. Will be removed in IntelMQ version 4.0.0, use *reports* instead.
 * `apikey`: Your Shadowserver API key
 * `secret`: Your Shadowserver API secret
+* `reports`: A list of strings or a comma-separated list of the mailing lists you want to process.
```

Comment:
What is a "mailing list" in this context?

Reply:
Mailing lists are how reports were delivered before API delivery was available.  Some organizations had multiple mailing lists based on a filter type (e.g. geo and IP/CIDR).